lib/test-worker.js

```diff
@@ -86,6 +86,12 @@ var oldNodeModulesPaths = module.constructor._nodeModulePaths;
 module.constructor._nodeModulePaths = function () {
 	var ret = oldNodeModulesPaths.apply(this, arguments);
 	ret.push(nodeModulesDir);
+	if (process.env.NODE_PATH) {
+		var osSplitChar = /^win/.test(process.platform) ? ';' : ',';
```

Comment:
I didn't want to add another dependency, but if it's fine for you I think too that this would be a better solution.